analyzer/codechecker_analyzer/logger.py

```diff
@@ -0,0 +1,155 @@
+# -------------------------------------------------------------------------
```

Comment:
Why did you copy this file in this pull request? You don't use it for now.

Reply:
Yes, this should not be here I started to cleanup the logging too and added this by accident.